htdocs/js/PGProblemEditor/pgproblemeditor.js

```diff
@@ -138,10 +138,41 @@
 			.catch((err) => showMessage(`Error: ${err?.message ?? err}`));
 	};
 
+	// Send a request to the server to convert_to_PGML the current PG code in the CodeMirror editor.
```

Comment:
I just noticed this comment.  Please change this to something like
```suggestion
	// Send a request to the server to convert the current PG code in the CodeMirror editor to PGML.
```
Comments should not refer to some non-existent method like `convert_to_PGML`.